### PR TITLE
added board member/retired staff description

### PR DESCRIPTION
**Pull request by lima0**
added board member/retired staff description in the faq page

### CODE REVIEW

File: src/content/site/faq.md [hunk 1]
```diff
@@ -20,6 +20,10 @@ This role recognizes people who contribute their time and knowledge to the serve
 
 ChatMod/Moderator/Senior Moderator/Admin/Owner - These people ensure that the usage of this environment reflects the guidelines as illustrated in the rules. Staff is occasionally handpicked, and sometimes there is an application process. There is a trial period for all new staff of 2-4 weeks (sometimes more, sometimes less) which comes with a trial role, however, they are treated as full staff members during this time. Please respect all staff equally; if you have an issue with a specific staff member, talk to an administrator.
 
+Senior Board Member/Board Member - This is for staff members who are currently not active moderators.
```
Comment (ilitr) [inline src/content/site/faq.md:23]: Do we need these two roles to be described?

Comment (lima0) [reply src/content/site/faq.md:23]: i asked in #server-meta before making this PR

Comment (Moe-Szyslak) [reply src/content/site/faq.md:23]: Might be worth also adding Administrative Board Member to this list

Comment (lima0) [reply src/content/site/faq.md]: done 👍

**Review by Moe-Szyslak — APPROVED** [review]

**Pull request merged**